src/icons/CCLicenseIcon.tsx

```diff
@@ -0,0 +1,46 @@
+import { SxProps } from '@mui/material';
+import Box from '@mui/material/Box';
+
+import React, { FC } from 'react';
+
+import { CCLicenseAdaption } from '../constants';
+
+type Props = {
+  adaption: CCLicenseAdaption;
+  sx?: SxProps;
+  alt?: string;
+};
+
+const CCLicenseIcon: FC<Props> = ({
+  adaption,
+  sx,
+  alt = 'Creative Commons License',
+}) => {
```

Comment:
Is there a nicer version of this icon? I'm not sure I like how it looks.

Reply:
These are the official icons.